.github/workflows/package-test.yml

```diff
@@ -34,4 +34,6 @@ jobs:
           python -m playwright install
 
       - name: Run Tests
+        env:
+          PROXY: ${{ secrets.PROXY }}
```

Comment:
does this mean we'll always need to have a proxy ready?

Reply:
Not necessarily, but you might if you're making a lot of requests. I'm on university internet and keep getting blocked by captchas so I seem to need to use them.

Reply:
Hmm, that's interesting. I've been starting to see captchas appearing on the website, but only in firefox. I'm on a residential connection in Canada though, so we might be seeing different things because of that. Good to know!